### PR TITLE
[#11572] Notification Feature - Add test for GetNotificationAction and fix comments

### PR DESCRIPTION
**Pull request by ziqing26**
Part of #11572 

**Outline of Solution**

Add tests for `GetNotificationAction` and address comments in #11711 
* Add `transient` keyword to updatedAt in `NotificationAttribute`.
* Fix alignment in html file of edit component.
* Refactor `GetNotificationAction`.
* Remove not used function.

### CODE REVIEW

File: src/test/java/teammates/ui/webapi/GetNotificationActionTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        loginAsAdmin();
+    }
+
+    @Test
+    @Override
+    protected void testAccessControl() {
+        verifyOnlyAdminCanAccess();
+    }
+
+    @Test
```
Comment (NicolasCwy) [inline src/test/java/teammates/ui/webapi/GetNotificationActionTest.java:36]: This test annotation can be omitted since no test method are executed here

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
LGTM just one nit

File: src/test/java/teammates/ui/webapi/GetNotificationActionTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        GetNotificationAction action = getAction(requestParams);
+        JsonResult jsonResult = getJsonResult(action);
+
+        NotificationData output = (NotificationData) jsonResult.getOutput();
+        verifyNotificationEquals(expected, output);
+    }
+
+    @Test
+    protected void testExecute_withInvalidNotificationId_shouldThrowError() {
+        ______TS("Failure: Notification id does not exist.");
```
Comment (jianhandev) [inline src/test/java/teammates/ui/webapi/GetNotificationActionTest.java:60]:
```suggestion
        ______TS("Failure: Notification does not exist");
```

File: src/test/java/teammates/ui/webapi/GetNotificationActionTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+        GetNotificationAction action = getAction(Const.ParamsNames.NOTIFICATION_ID, "invalid-notif");
+        EntityNotFoundException enfe = assertThrows(EntityNotFoundException.class, action::execute);
+        assertEquals("Notification does not exist.", enfe.getMessage());
+
+        ______TS("Failure: Notification id is null.");
```
Comment (jianhandev) [inline src/test/java/teammates/ui/webapi/GetNotificationActionTest.java:66]:
```suggestion
        ______TS("Failure: Notification id is null");
```

File: src/test/java/teammates/ui/webapi/GetNotificationActionTest.java (hunk 1 above)
Comment (ziqing26) [reply src/test/java/teammates/ui/webapi/GetNotificationActionTest.java:36]: Okay thank you!

**Review by jianhandev — APPROVED** [review]
LGTM

**Review by NicolasCwy — APPROVED** [review]
LGTM!

**Review by jianhandev — APPROVED** [review]

**Review by t-cheepeng — APPROVED** [review]

**Pull request merged**